server/player/player.go

```diff
@@ -1718,25 +1720,30 @@ func (p *Player) placeBlock(pos cube.Pos, b world.Block, ignoreBBox bool) bool {
 
 // obstructedPos checks if the position passed is obstructed if the block passed is attempted to be placed.
 // The function returns true if there is an entity in the way that could prevent the block from being placed.
-func (p *Player) obstructedPos(pos cube.Pos, b world.Block) bool {
+func (p *Player) obstructedPos(pos cube.Pos, b world.Block) (bool, bool) {
 	w := p.World()
 	blockBoxes := b.Model().BBox(pos, w)
 	for i, box := range blockBoxes {
 		blockBoxes[i] = box.Translate(pos.Vec3())
 	}
 
+	resend := true
```

Comment:
instead of having this here, I would have the returned type be name `(obstructed bool, resend bool)`